### DISCUSSION TITLE
Merging Groups in Contacts

### DISCUSSION THREAD

**Discussion by Corvin4** [FossifyOrg/General-Discussion#247] [Ideas]
Hi, I'm currently trying out the Contacts app. I would be happy if groups with the same name could be merged in the Groups tab in the same way as in the individual contacts.

**Comment by Aga-C** [comment]
Please create a feature request in [Contacts](https://github.com/FossifyOrg/Contacts/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.
